model-optimizer/mo/graph/graph.py

```diff
@@ -816,7 +818,7 @@ def _dump_nodes_attrs():
                 attrs = self.node[node_id]
                 color = fill_color_by_type.get(attrs.get('type', ''), fill_color[attrs['kind']])
 
-                if node_id in highlight_nodes or 'highlight' in node_attrs and node_attrs['highlight']:
+                if node_id in highlight_nodes or 'highlight' in node_attrs and fill_color['highlight']:
```

Comment:
The `fill_color['highlight']` is always defined so there is no need to check for it here. The idea of the check was to make sure that attribute `highlight` is defined in the node attributes and is True. So I don't agree with your changes. The problem in the existing check that there are missing parentheses and because of this the logic is broken. The code should look like this:

```python
if node_id in highlight_nodes or ('highlight' in node_attrs and node_attrs['highlight']):
```

or it can be simplified even more:

```python
if node_id in highlight_nodes or node_attrs.get('highlight'):
```

Reply:
Node attrs in this local function scope was not a map but rather a list that meant **node attrs to be visualized**. 
https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/mo/graph/graph.py#L772-L773

While in other places node_attrs is a dict that meant a dictionally with node attrs
https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/mo/graph/graph.py#L787-L799

I was changing to enhance visualizing node types, but anyways I reverted back.

Reply:
Ok. I see. So let's change this to:
```python
if node_id in highlight_nodes or attrs.get('highlight'):
    color = fill_color['highlight']
```